eth-contracts/contracts/ForceBridge.sol

```diff
@@ -32,6 +33,12 @@ contract ForceBridge {
     bytes32 private _HASHED_VERSION;
     bytes32 private _TYPE_HASH;
 
+    mapping(uint256 => bool) private unlockUsedNonces;
```

Comment:
no need to maintain this mapping, just compare nonce in parameter with latestUnlockNonce_